src/types.ts

```diff
@@ -190,7 +190,6 @@ const ItemQuota = t.intersection([
   t.partial({
     quotaRefreshTime: t.number,
     transactionTime: DateFromNumber,
-    identifierInputs: t.array(IdentifierInput),
```

Comment:
Is there a corresponding backend PR to remove this from quota response?

Reply:
Not at the moment, will add a separate task for it. Am thinking that it's not as high priority since it doesn't need to be in place for the return flow to work.

Reply:
As long as we are tracking it somewhere, else it might cause future confusion :)